src/rgw/radosgw-admin/radosgw-admin.cc

```diff
@@ -9100,6 +9123,78 @@ int main(int argc, const char **argv)
     }
   }
 
+  if (opt_cmd == OPT::DEDUP_STATS) {
+    using namespace rgw::dedup;
+    rgw::sal::RadosStore *store = dynamic_cast<rgw::sal::RadosStore*>(driver);
+    if (!store) {
+      cerr << "ERROR: this command can only work when the cluster has a RADOS "
```

Comment:
s/the cluster/RGW/

Reply:
This exact error message is used all over the file for this scenario (I counted 10 places using this error message)